polaris-core/src/main/java/org/apache/polaris/core/PolarisConfiguration.java

```diff
@@ -167,12 +168,11 @@ public static <T> Builder<T> builder() {
           .key("SUPPORTED_CATALOG_STORAGE_TYPES")
           .catalogConfig("supported.storage.types")
           .description("The list of supported storage types for a catalog")
-          .defaultValue(
-              List.of(
+          .defaultValue(new ArrayList<>(List.of(
```

Comment:
Since the PolarisConfiguration is of the type `List`, can we push the wrapping down into the `defaultValue` method?

Reply:
Thanks for review. Please take another look when you get a chance to see if that is what you are looking for.